src/Form/formatters.js

```diff
@@ -78,7 +78,7 @@ const getCountryName = (el, options = {}) => {
     if (!addressCountryCode) return ''
 
     // Try to infer the field language or fallback to en
-    const elLocale = inferElementLocale(el)
+    const elLocale = inferElementLocale(el) || 'en'
```

Comment:
Probably not one to change for this small change PR, but it might be nicer for `inferElementLocale` to return `undefined` when there's no locale to infer, and the consuming functions (in this case `getCountryDisplayName`) can decide on the fallback language (and saved us adding `|| 'en'` or `|| 'unknown'` around the place).

Reply:
Mhm, I think that's more or less what this change is doing. `inferElementLocale` now returns empty string instead of `'en'` (see the diff above), then the consumers apply the fallbacks they need. I'm not using `undefined` to keep the same type. Let me know if I missed your point.